### PR TITLE
Deployment: Base network

### PR DESCRIPTION
**Pull request by EndymionJkb**
# Deployment: Base network

## Deployment tasks checklist:

- [X] Contract(s) deployed to all target networks / testnets
- [X] TX IDs are properly generated
- [X] Action IDs are generated after deployment (if applicable) 
- [X] Outputs for all target networks are generated, and linked in the task `readme`
- [X] Addresses files are up to date with the new deployment 
- [X] Contracts are verified in every network 
- [N/A] Deployments `CHANGELOG` is updated
- [X] There are no code changes of any kind

### CODE REVIEW

File: .github/workflows/deployment-checks.yml [hunk 1]
```diff
@@ -97,6 +97,16 @@ jobs:
...
+  check-base-deployments:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Check Base Deployment Addresses
+        uses: ./.github/actions/check-network-deployments
+        env:
+          BASE_RPC_ENDPOINT: ${{ secrets.BASE_RPC_ENDPOINT }}
+        with:
+          network-name: base
```
Comment (jubeira) [inline .github/workflows/deployment-checks.yml:109]: We need to add the check for action IDs as well (see below for examples).

File: addresses/.supported-networks.json [hunk 2]
```diff
@@ -34,5 +34,9 @@
   "zkevm": {
     "chainId": 1101,
     "block-explorer": "https://zkevm.polygonscan.com"
+  },
+  "base": {
+    "chainId": 8453,
+    "block-explorer": "https://wwstage.basescan.org"
```
Comment (jubeira) [inline addresses/.supported-networks.json:40]: isn't it https://basescan.org/?

File: package.json
Comment (jubeira) [inline package.json:1]: micronit: maybe I'd suggest keeping the order of the networks in the command as we do in the readme files. Following that logic, base goes after zkevm and before goerli.

File: package.json [hunk 3]
```diff
@@ -25,12 +25,12 @@
...
     "check-network-action-ids": "hardhat check-action-ids --network",
-    "check-action-ids": "hardhat check-action-ids --network mainnet && hardhat check-action-ids --network polygon && hardhat check-action-ids --network arbitrum && hardhat check-action-ids --network optimism && hardhat check-action-ids --network bsc && hardhat check-action-ids --network avalanche && hardhat check-action-ids --network zkevm && hardhat check-action-ids --network sepolia",
+    "check-action-ids": "hardhat check-action-ids --network mainnet && hardhat check-action-ids --network polygon && hardhat check-action-ids --network arbitrum && hardhat check-action-ids --network optimism && hardhat check-action-ids --network bsc && hardhat check-action-ids --network avalanche && hardhat check-action-ids --network zkevm && hardhat check-action-ids --network sepolia && hardhat check-action-ids --network base",
```
Comment (jubeira) [inline package.json:30]: I've just realized we don't have goerli here. Probably not a big deal, but we should add it afterwards.

**Review by jubeira — DISMISSED** [review]
Great work! Just a few minor comments, but otherwise looks good!

File: addresses/.supported-networks.json (hunk 2 above)
Comment (EndymionJkb) [reply addresses/.supported-networks.json:40]: I guess it is now that it seems to be live :) Did this when it wasn't.

File: .github/workflows/deployment-checks.yml [hunk 4]
```diff
@@ -122,7 +132,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
-      - name: Check Mainnet Deployment Addresses
+      - name: Check Mainnet Action IDs
```
Comment (EndymionJkb) [inline .github/workflows/deployment-checks.yml:135]: Could do this in a separate PR, if we're being strict

Comment (jubeira) [reply .github/workflows/deployment-checks.yml:135]: ah, thanks; I think it's fine.

**Review by jubeira — APPROVED** [review]
:ship:
